tests/unittests/benchmark/task/profet/test_profet_task.py

```diff
@@ -0,0 +1,455 @@
+""" Utilities used to test the different subclasses of the ProfetTask. """
+from dataclasses import asdict, replace
+from logging import getLogger as get_logger
+from pathlib import Path
+from typing import Any, Callable, ClassVar, Dict, Type
+
+import numpy as np
+import pytest
+import torch
+from orion.algo.space import Space, _Discrete
+from orion.benchmark.task.profet.profet_task import MetaModelConfig, ProfetTask
+from _pytest.monkeypatch import MonkeyPatch
+from _pytest.tmpdir import TempPathFactory
+
+from .conftest import REAL_PROFET_DATA_DIR, c_max, c_min, is_nonempty_dir, y_max, y_min
+
+logger = get_logger(__name__)
+
+_devices = ["cpu"]
+if torch.cuda.is_available():
+    _devices.append("cuda")
+
+
+class ProfetTaskTests:
+    """Base class for testing Profet tasks."""
+
+    Task: ClassVar[Type[ProfetTask]]
+
+    @pytest.fixture()
+    def profet_train_config(self):
+        """Fixture that provides a configuration object for the Profet algorithm for testing."""
+        quick_train_config = self.Task.ModelConfig(
+            task_id=0,
+            seed=123,
+            num_burnin_steps=10,
+            # num_steps=10,
```

Comment:
So we keep the large default value for num_steps? We could either set it here or remove the comment.

Reply:
Fixed, set a value in db4f745